configs/experiment/birdset_neurips24/HSN/DT/efficientnet.yaml

```diff
@@ -8,12 +9,12 @@ defaults:
   - override /datamodule/transforms: bird_default_multilabel.yaml
   - override /paths: default.yaml
   - override /hydra: default.yaml
-
```

Comment:
tags should only by ["birdsetDT"]? why did you change it?

Reply:
@lurauch I like more tags better. I guess it is not relevant to change the tags.